src/backend/views/components/navigator.twig

```diff
@@ -0,0 +1,52 @@
+{#
+Reusable navigaor component.
+Available props:
+  - previousPage
+  - nextPage
+  - class: additional class for the navigator
+
+Usage example:
+  {% include 'components/navigator.twig' with {previousPage: previousPage, nextPage: nextPage} %}
+#}
+
+{% set mainClass = 'navigator__item' %}
+
+
+{% set tag = 'div' %}
+
+{% set tag = 'a' %}
+
+<div>
```

Comment:
useless wrappers

Reply:
No, it uses for showing next page navigator on the right side if there is no previous page button, because page__footer has space-between parameter